readme.md

```diff
@@ -807,6 +807,7 @@
 - [Humane Technology](https://github.com/humanetech-community/awesome-humane-tech#readme) - Open source projects that help improve society.
 - [Mental Health](https://github.com/dreamingechoes/awesome-mental-health#readme) - Mental health awareness and self-care in the software industry.
 - [Neuroscience](https://github.com/analyticalmonk/awesome-neuroscience#readme) - Study of the nervous system and brain.
+- [Decentralized Science](https://github.com/DeSciWorldDAO/awesome-desci#readme) - Descentralized Science: Science using blockchain technology to solve issues related to funding, publishing, data validation and sharing.
```

Comment:
Looks good @gingerheart86! I'm a big DeSci fan myself, thanks for adding this!

 Just a few small comments:
The link text already says Decentralized Science, you don't need to mention it again in the description. I also made the description a bit more gramatically correct and concise.

```suggestion
- [Decentralized Science](https://github.com/DeSciWorldDAO/awesome-desci#readme) - Use blockchain technology to develop, fund and share science.
```

Reply:
hey! I liked your suggestion! thanks! I'm not native english speaker so grammar corrections are most welcome!
I think I might change blockchain for DLT